charts/vela-core/templates/defwithtemplate/suspend.yaml

```diff
@@ -12,8 +12,20 @@ spec:
   schematic:
     cue:
       template: |
+        import (
+        	"vela/op"
+        )
+
+        suspend: op.#Suspend & {
+        	if parameter.duration != _|_ {
+        		duration: parameter.duration
+        	}
+        	if parameter.message != _|_ {
+        		message: parameter.message
+        	}
+        }
         parameter: {
-        	// +usage=Specify the wait duration time to resume workflow such as "30s", "1min" or "2m15s"
         	duration?: string
```

Comment:
does ·duration· need default value？

Reply:
I don't think duration needs a default value, it should be decided by the user.

Reply:
Why did we remove the description?